config/routes.rb

```diff
@@ -12,7 +12,8 @@
 
   resources :categories, only: [:index]
   resources :items, only: [:index, :show]
-  resource :cart, only: [:show, :delete, :edit, :update, :create]
+  resource :cart, only: [:show, :destroy, :create]
+  patch '/items/:id' => 'carts#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.